content/python/concepts/strings/terms/lower/lower.md

```diff
@@ -67,3 +67,17 @@ if my_string.lower() == "amazing!":
 
 # Output: "Isn't that just AMAZING!""
 ```
+
+## Codebyte Example
+
+Comparing `my_favorite_color` to `user_favorite_color` using `.lower()`:
+
+```codebyte/python
+my_favorite_color = "Blue"
+user_favorite_color = input("What is your favorite color? : ")
```

Comment:
the input() function isn't supported by the Codecademy compiler so this example will throw an error. Please revise the example. Additionally, note that docs code uses 2 spaces, not tabs. The use of tabs will return file errors.